### PR TITLE
Add donations and support chat information

### PR DESCRIPTION
**Pull request by Echolon**
Test text 1

- test
- liste
- hier

### CODE REVIEW

File: ReadMe.md [hunk 1]
```diff
@@ -4,23 +4,42 @@
 [![alpha.build-push](https://github.com/monal-im/Monal/actions/workflows/alpha.build-push.yml/badge.svg?branch=alpha.build)](https://github.com/monal-im/Monal/actions/workflows/alpha.build-push.yml)
 [![Translation status](https://hosted.weblate.org/widgets/monal/-/svg-badge.svg)](https://hosted.weblate.org/engage/monal/?utm_source=widget)
 
-This is the Monal XMPP client as found in the app store.  If you want to  use the latest stable versions, search for Monal in the iOS or OSX app store. Visit my blog to read about the development: www.monal.im
+This is the Monal XMPP client as found in the app store.  If you want to  use the latest stable versions, search for Monal in the iOS or OSX app store. Visit my blog to read about the development: 
 
-You can get the current releases in the App Store;
+### www.monal.im
```
Comment (tmolitor-stud-tu) [inline ReadMe.md]: that is not a link but a heading!

File: ReadMe.md [hunk 2]
```diff
@@ -4,23 +4,42 @@
...
 ### Impressions from the MacOS version
 
-<img src="https://monal.im/wp-content/uploads/2020/02/Screen-Shot-2020-02-15-at-10.28.38-PM-1.png" width="600">
+<img src="https://monal.im/wp-content/uploads/2020/02/Screen-Shot-2020-02-15-at-10.28.38-PM-1.png" width="500">
+
+## Support Chat (MUC)
+
+You can join this public chat (MUC) via XMPP:
+
+## monal@chat.yax.im
```
Comment (tmolitor-stud-tu) [inline ReadMe.md]: same a above...make this a link...ideally a ?join xmpp link (see the corresponding xep for the exact syntax)

File: ReadMe.md [hunk 3]
```diff
@@ -4,23 +4,42 @@
...
+
+## Donations and Support
+
+Monal is developed by volunteers and community collaboration. The work which has been done is usually not paid and the developers ask for donations to keep up service costs and development in the future! Please consider to give a little bit back for the hard work which has been conducted. Currently there are three ways for financial support of the Monal development:
+
+- Donate via [GitHub Sponsors](https://github.com/sponsors/tmolitor-stud-tu)
+- Donate via [Libera Pay](https://liberapay.com/tmolitor)
+- For EU citizens you can ask the respective core developer for the IBAN number. Please contact Thilo Molitor via mail to thilo+monal@eightysoft.de
```
Comment (tmolitor-stud-tu) [inline ReadMe.md]: I wrote a better EU sentence

File: ReadMe.md (hunk 1 above)
Comment (Echolon) [reply ReadMe.md]: Yes, I wanted to increase the font size

**Comment by Echolon** [comment]
Anything blocking a merge, apart from rebase? @tmolitor-stud-tu

**Comment by tmolitor-stud-tu** [comment]
No

Am 22. Mai 2021 13:36:59 MESZ schrieb Echolon ***@***.***>:

**Comment by tmolitor-stud-tu** [comment]
git fetch origin
git checkout Echolon-update-ReadMe-20210513
git reset --hard origin/Echolon-update-ReadMe-20210513
git rebase -i origin/develop  # delete all commits in this list not coming from you
git push --force

File: ReadMe.md (hunk 1 above)
Comment (tmolitor-stud-tu) [reply ReadMe.md]: No, make this a link so that it is clickable

File: ReadMe.md [hunk 4]
```diff
@@ -4,23 +4,42 @@
...
 ### Impressions from the MacOS version
 
-<img src="https://monal.im/wp-content/uploads/2020/02/Screen-Shot-2020-02-15-at-10.28.38-PM-1.png" width="600">
+<img src="https://monal.im/wp-content/uploads/2020/02/Screen-Shot-2020-02-15-at-10.28.38-PM-1.png" width="500">
+
+## Support Chat (MUC)
+
+You can join this public chat (MUC) via XMPP:
+
+## xmpp:monal@chat.yax.im?join
```
Comment (tmolitor-stud-tu) [inline ReadMe.md]: Same here, make this a clickable link

File: ReadMe.md [hunk 5]
```diff
@@ -4,23 +4,42 @@
...
+## Donations and Support
+
+Monal is developed by volunteers and community collaboration. The work which has been done is usually not paid and the developers ask for donations to keep up service costs and development in the future! Please consider to give a little bit back for the hard work which has been conducted. Currently there are three ways for financial support of the Monal development:
+
+- Donate via [GitHub Sponsors](https://github.com/sponsors/tmolitor-stud-tu)
+- Donate via [Libera Pay](https://liberapay.com/tmolitor)
+- EU citizens can donate via SEPA, too. Just contact Thilo Molitor via mail to thilo+monal@eightysoft.de to get his IBAN.
+
+Here you can read about further [support of the development](https://github.com/monal-im/Monal/issues/363)!
+Find general information in the [Monal Wiki](https://github.com/monal-im/Monal/wiki).
```
Comment (tmolitor-stud-tu) [inline ReadMe.md:41]: Make sure there is a line break between these two lines (search Wikipedia on how to accomplish line breaks with markdown)

**Review by tmolitor-stud-tu — CHANGES_REQUESTED** [review]

File: ReadMe.md [hunk 6]
```diff
@@ -4,23 +4,41 @@
 [![alpha.build-push](https://github.com/monal-im/Monal/actions/workflows/alpha.build-push.yml/badge.svg?branch=alpha.build)](https://github.com/monal-im/Monal/actions/workflows/alpha.build-push.yml)
 [![Translation status](https://hosted.weblate.org/widgets/monal/-/svg-badge.svg)](https://hosted.weblate.org/engage/monal/?utm_source=widget)
 
-This is the Monal XMPP client as found in the app store.  If you want to  use the latest stable versions, search for Monal in the iOS or OSX app store. Visit my blog to read about the development: www.monal.im
+This is the Monal XMPP client as found in the app store.  If you want to  use the latest stable versions, search for Monal in the iOS or OSX app store. [Visit the blog to read about the development](www.monal.im)!
```
Comment (tmolitor-stud-tu) [inline ReadMe.md]: please make this link complete (including https)

File: ReadMe.md [hunk 7]
```diff
@@ -4,23 +4,41 @@
...
 ### Impressions from the MacOS version
 
-<img src="https://monal.im/wp-content/uploads/2020/02/Screen-Shot-2020-02-15-at-10.28.38-PM-1.png" width="600">
+<img src="https://monal.im/wp-content/uploads/2020/02/Screen-Shot-2020-02-15-at-10.28.38-PM-1.png" width="500">
+
+## Support Chat (MUC)
+
+You can join this public chat (MUC) via XMPP:
+
+xmpp:monal@chat.yax.im?join
```
Comment (tmolitor-stud-tu) [inline ReadMe.md]: please make this a link:
`[monal@chat.yax.im](xmpp:monal@chat.yax.im?join)`

**Review by tmolitor-stud-tu — CHANGES_REQUESTED** [review]

**Review by tmolitor-stud-tu — APPROVED** [review]

**Pull request merged**